presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -30,6 +30,7 @@
 import com.facebook.presto.spi.security.GrantInfo;
```

Comment:
I think I am missing tests like `TestLogicalPlanner` to see what plans are generated for CREATE and INSERT queries

Reply:
I was thinking about such a test, but finally i decided to do not implement one. Statistic aggregations don't change the plan shape but just add additional output symbols. 

Currently `BasePlanTest` uses the `LocalQueryRunner` with the TPCH connector installed. TPCH connector doesn't ask the engine to compute any statistics. So to do that i would have to change the TPCH connector to request statistics for some tables. Also i would need to have additional tables that request statistics to be pre-grouped. 

I found that test to be too tedious for the value it introduces. What this test can actually verify, is that the planner creates the correct aggregations for given column-statistics pairs. The regression caused by modifying that code will be caught by the product/integration tests that have all possible aggregation for the all possible types covered. We just may spend a little bit longer trying to figure out what has happened.